documentation/src/docs/rest/restapis.md

```diff
@@ -489,20 +481,10 @@ List streams within the given scope
 ```
 
 
-##### Request query
```

Comment:
why is this section removed?

Reply:
This was auto-generated by the swagger2markup-cli-1.3.3.jar.  Also, we can observe that the query parameter for the REST api has been documented @ https://github.com/shrids/pravega/blob/issue-fixRESTdocumentation/documentation/src/docs/rest/restapis.md#parameters-6